tapfreighter/parcel.go

```diff
@@ -464,6 +471,9 @@ type sendPackage struct {
 	// TransferTxConfEvent contains transfer transaction on-chain
 	// confirmation data.
 	TransferTxConfEvent *chainntnfs.TxConfirmation
+
+	// Note is a user provided description for this transfer.
```

Comment:
nit: add something to the effect of "this is currently only used for burns", since we don't store it in the transfer table currently.